index.html

```diff
@@ -0,0 +1,767 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Building with Responsive Design</title>
+    <script src="https://kit.fontawesome.com/e252d07dd4.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <div class="top-nav grid">
+        <div class="news d-flex">
+          <div class="nav-item items1">
+            <a class="nav_a" href="#">News</a>
+          </div>
+          <div class="nav-item items2">
+            <a class="nav_a" href="#">Events</a>
+            <i class="fas fa-sort-down"></i>
+          </div>
+          <div class="nav-item items3">
+            <a class="nav_a" href="#">Business</a>
+            <i class="fas fa-sort-down"></i>
+          </div>
+          <div class="nav-item items4"><a class="nav_a" href="#"></a></div>
+          <div class="nav-item items5">
+            <a class="nav_a" href="#">Spaces</a>
+          </div>
+          <div class="nav-item items6">
+            <a class="nav_a" href="#">Terms & Conditions</a>
+          </div>
+        </div>
+        <div class="about d-flex">
+          <div class="nav-item items1"><a class="nav_a" href="#">About</a></div>
+          <div class="nav-item items2">
+            <a class="nav_a" href="#">Advertise</a>
+          </div>
+          <div class="nav-item items3"><a class="nav_a" href="#">Jobs</a></div>
+          <div class="nav-item items4">
+            <a class="nav_a" href="#">Contact</a>
```

Comment:
**On Small Screens**

- [ ] The Black Bar should disappear

| Yours | Original |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/11758151/93010240-3f9c8f80-f582-11ea-84c8-38ebf0ed1831.png) | ![image](https://user-images.githubusercontent.com/11758151/93010244-4c20e800-f582-11ea-90a9-4a061053a2ef.png) |